Tests/Common/Statistics/AnnualPerformanceTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using System.Linq;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace QuantConnect.Tests.Common.Statistics
+{
+    [TestFixture]
+    public class AnnualPerformanceTests
+    {
+        [Test]
+        public void ZeroTradingDays()
```

Comment:
Please write a test against the SPY daily test data included in LEAN with realistic daily performances and realistic annual performance. 

You can reuse the same array of SPY performance data to cherry pick periods with high returns and test better annual performances. Use a [TestData] notation on the test method to pass in a range of dates and expected results.